src/services/ui/src/pages/actions/index.tsx

```diff
@@ -14,6 +14,39 @@ import {
 } from "@/components";
 import { ModalProvider } from "@/pages/form/modals";
 import { detailsAndActionsCrumbs } from "@/pages/actions/actions-breadcrumbs";
+import {
+  chipRespondToRaiSetup,
+  chipWithdrawPackageSetup,
+  defaultIssueRaiSetup,
+  defaultWithdrawRaiSetup,
+  FormSetup,
+  medicaidRespondToRaiSetup,
+  medicaidWithdrawPackageSetup,
+} from "@/pages/actions/setups";
+
+type SetupOptions = "CHIP SPA" | "Medicaid SPA";
```

Comment:
Typing right now on our transforms are dodgy, and `PlanType` does not correlate with the values given back in the `planType` object property. I think our continued type refactors will be remedying some of this, so for now I've plugged in this type here for use as needed.